src/javascript/binary/common_functions/valid_account_opening.js

```diff
@@ -57,7 +57,7 @@ var ValidAccountOpening = (function(){
       errorFname.innerHTML = Content.errorMessage('min', '2');
       Validate.displayErrorMessage(errorFname);
       window.accountErrorCounter++;
-    } else if (!/^[a-zA-Z\s-.']+$/.test(fname.value)){
+    } else if (/[`~!@#$%^&*)(_=+\[}{\]\\\/";:\?><,|\d]+/.test(fname.value)){
```

Comment:
We can't use that regex in javascript. As an alternative way, this regex invalidates unwanted characters and allow others.

This is exactly same as what is used for Japanese clients here: https://github.com/binary-com/binary-static/blob/master/src/javascript/binary/websocket_pages/user/new_account/japan_acc_opening/japan_acc_opening.ui.js#L105